angularjs/manageVariable/model.js

```diff
@@ -215,6 +215,8 @@
             });
 
             delete variable.typeMetadata;
+            delete variable.created_date_pretty;
+            delete variable.updated_date_pretty;
```

Comment:
@AltamashShaikh can we do the same for `created_date` and `updated_date` and `status`. These don't seem to be used by the API either

Reply:
@tsteur Added